stratum/hal/lib/common/gnmi_publisher.cc

```diff
@@ -114,6 +114,7 @@ ::util::Status GnmiPublisher::HandlePoll(const SubscriptionHandle& handle) {
   if ((status = (*handle)(PollEvent())) != ::util::OkStatus()) {
```

Comment:
^^^ HandleChange discards error as well

Check to see if there are other discarded statuses